src/m3/test/constraints.test.ts

```diff
@@ -0,0 +1,94 @@
+import {
+    assertEquals
+} from "https://deno.land/std@0.160.0/testing/asserts.ts"
+
+import {
+    Annotation,
+    Concept,
+    ConceptInterface,
+    Containment,
+    Metamodel,
+    Multiplicity,
+    Property,
+    Reference
+} from "../types.ts"
+import {issuesMetamodel} from "../constraints.ts"
+
+
+Deno.test("constraints (lioncore)", async (tctx) => {
```

Comment:
I think it would be important to guarantee compatibility with the boring but established Node.js

Reply:
A wrapper for Mocha (which is the most boring and established unit test framework in the Node.js ecology) exists: https://deno.land/x/deno_mocha@0.3.0
But I still don't haven't figured how to have sources that are Deno- and Node.js-compatible at the same time.